rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/nodesource/serialization/export/catalog/ExportToCatalogConfirmWindow.java

```diff
@@ -183,50 +194,50 @@ private void fillBucketList() {
-                                                                       nodeSourceNameInBucket + "/revisions",
+                                                                       catalogObjectName + "/revisions",
                                                                        new RequestCallback() {
                                                                            @Override
                                                                            public void onResponseReceived(
                                                                                    Request request, Response response) {
-                                                                               JSONArray nodeSourceRevisions = JSONParser.parseStrict(response.getText())
-                                                                                                                         .isArray();
-                                                                               JSONObject lastNodeSourceRevision = nodeSourceRevisions.get(0)
-                                                                                                                                      .isObject();
+                                                                               JSONArray catalogObjectRevisions = JSONParser.parseStrict(response.getText())
+                                                                                                                            .isArray();
+                                                                               JSONObject lastCatalogObjectRevision = catalogObjectRevisions.get(0)
+                                                                                                                                            .isObject();
                                                                                revisionLabel.setContents("(Last Revision: " +
-                                                                                                         lastNodeSourceRevision.get("commit_time")
-                                                                                                                               .isString()
-                                                                                                                               .stringValue() +
+                                                                                                         lastCatalogObjectRevision.get("commit_time")
+                                                                                                                                  .isString()
+                                                                                                                                  .stringValue() +
                                                                                                          " \"" +
```

Comment:
why there is space before back-slash?

Reply:
It is needed to separate commit time from commit message.